deploy/kblib/values.yaml

```diff
@@ -26,3 +26,7 @@ hostNetworkAccessible: false
 ## @param publiclyAccessible if true, expose the internet service
 ##
 publiclyAccessible: false
+
+## @param rbacEnabled if true, will create and use RBAC resources
+##
+rbacEnabled: false
```

Comment:
for user that only work with Cluster Helm chart, should this be true as default?

Reply:
The default rbac permission of the controller is true, so I think the default false here is appropriate.

Reply:
Set to true [0f15791](https://github.com/apecloud/kubeblocks/pull/4257/commits/0f157914c9465a3a13f518641f0a96c74bac1db8)